lib/active_record/connection_adapters/clickhouse/schema_creation.rb

```diff
@@ -22,7 +22,9 @@ def add_column_options!(sql, options)
 
         def add_table_options!(create_sql, options)
           is_view = create_sql.match(/^CREATE\s+(MATERIALIZED\s+)?VIEW/)
```

Comment:
Is this regex working with `CREATE LIVE VIEW`?

Reply:
kinda, for how it works the function that call `add_table_options!` even for a `LIVE VIEW` the create_sql string match.
(Logically it shouldn't).
The other option is to change more rows and add variables to manage the `LIVE VIEW` as `MATERIALIZED VIEW`